sources/packages/web/src/views/aest/student/applicationDetails/AssessmentsSummary.vue

```diff
@@ -135,6 +147,11 @@ export default defineComponent({
       });
     };
 
+    const reloadComponents = () => {
+      historyKey.value++;
+      manualReassessmentKey.value++;
+    };
```

Comment:
I'm not sure about this method. It seems unnecessary. Can you briefly explain what it does or leave a comment for the method?

Reply:
We need to reload the history component after creating a reassessment to display an updated list of assessments. One way to make it reload is to update the `:key` prop of the component.